### PR TITLE
Tab UI tab styling

### PR DESCRIPTION
**Pull request by robertdown**
This improves the tab layout

### CODE REVIEW

File: interface/themes/tabs_style_compact.css [hunk 1]
```diff
@@ -29,11 +29,12 @@ html
 }
 body
```
Comment (bradymiller) [inline interface/themes/tabs_style_compact.css]: Did you mean to modify this file or tabs_style_full.css ?

Was hoping to keep Ray's theme intact and just get the switch from div to a working for it.

If want to create another more compact modern theme (like your full), then sounds like we may need to create another theme file and change the names around  (rename Ray's to something else).

As with your full theme there are even more serious mouseover issues happening on this theme (difficulties in dragging the mouse down to the submenu). Have you been able to try your themes from a firefox browser in linux?

Comment (bradymiller) [inline interface/themes/tabs_style_compact.css]: btw,
I just realized I needed to empty my web browser cache before testing this. I'll do that and retest this.
-brady

Comment (robertdown) [inline interface/themes/tabs_style_compact.css]: Did not mean to edit the compact view

Comment (bradymiller) [inline interface/themes/tabs_style_compact.css]: That's why it looked a bit strange :)
The problem with converting from div to a is that it is breaking the compact view. Would this be an easy css fix?

**Comment by robertdown** [comment]
Yes, basically just a find and replace in the CSS… I’ll take a look tomorrow

Regards,
Robert

**Comment by bradymiller** [comment]
keeping this PR as reminder to change from div to a tags (need to ensure the compact tabs theme works with this)

**Pull request closed without merging**